vertical-pod-autoscaler/pkg/utils/test/test_utils.go

```diff
@@ -74,6 +74,16 @@ func Resources(cpu, mem string) apiv1.ResourceList {
 	return result
 }
 
+// AddResource add a resource to the given resource list
```

Comment:
I don't see where we are using this function, maybe we can remove it if it is not used.